Vagrantfile

```diff
@@ -6,17 +6,24 @@ NODES = 4
 DISKS = 8
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "chef/centos-7.1"
```

Comment:
If I comment the 'when' clause for "Setting up Network" task in common play, while using libvirt provider, the ``vagrant provision`` fails with the following error messages,

``
failed: [storage2] => {"changed": true, "cmd": ["ifup", "enp0s8"], "delta": "0:00:00.139595", "end": "2015-07-02 06:49:10.430911", "rc": 1, "start": "2015-07-02 06:49:10.291316", "warnings": []}
stderr: /sbin/ifup: configuration for enp0s8 not found.
``
Conclusion: We need the changes to the ansible script to make it work with libvirt provider.